hack/make-release-notes.sh

```diff
@@ -74,7 +74,7 @@ for f in "${download_assets[@]}"; do
 done
 echo
 echo "Thanks to our contributors for helping out with ${TAG}:"
-git log "$(git describe --tags --abbrev=0)..HEAD" --format=%an |
+git log "$(git describe --tags --match 'v*' --abbrev=0 "${TAG}^")..${TAG}" --format=%an |
```

Comment:
this is a bit interesting though:
```
git describe --tags --match 'v*' --abbrev=0 "v0.4.3^"
v0.4.1
```
it skips v0.4.2?

Reply:
ya im not sure why. that was the release i did so maybe i messed something up? although i do see it when i `git tag -l` and i see it here